src/Files.App/Views/Settings/GeneralPage.xaml.cs

```diff
@@ -1,15 +1,23 @@
 // Copyright (c) 2023 Files Community
 // Licensed under the MIT License. See the LICENSE.
 
+using Files.App.ViewModels.Settings;
+using Microsoft.UI.Xaml;
 using Microsoft.UI.Xaml.Controls;
 
 namespace Files.App.Views.Settings
 {
 	public sealed partial class GeneralPage : Page
 	{
+
 		public GeneralPage()
 		{
 			InitializeComponent();
 		}
+
+		private void DeleteTag(object sender, RoutedEventArgs e)
```

Comment:
Is this a copy and paste? It should probably be `RemoveStartupPage`.

Reply:
Whoops, it's not a copy paste but I happened to be looking at the tabs page right before adding that

Reply:
I've done this before 🙂